src/components/MyModal.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { StyleSheet, Modal, View } from 'react-native'
+import { Card, Paragraph } from 'react-native-paper'
+
+import { standardStyle } from '../assets/styles'
+import colors from '../assets/colors'
+import images from '../assets/images'
+
+
+
+class MyModal extends React.Component {
+  render () {
+    const {
+      text,
+      loader,
+      onPress,
+      visible,
+    } = this.props;
+
+    const cardCoverJSX = loader ? (
+      <Card.Cover
+        source={images.loader}
+        style={standardStyle.largeCardIcon}
+      />
+      ) : null;
+    
+    return (
+      <Modal
+        animationType="fade"
+        transparent={true}
+        visible={visible}
+        onRequestClose={() => onRequestClose}>
```

Comment:
This is just junk filler at the moment because we are not allowing the user to close the modal.
can set it to null as well I believe. Was just following an example.